wave_matching/tests/icp_tests.cpp

```diff
@@ -0,0 +1,80 @@
+// Tests for wrapper
+
+#include <gtest/gtest.h>
+#include <pcl/io/pcd_io.h>
+#include <Eigen/Dense>
+#include <iostream>
+
+#include "wave/matching/ICP.hpp"
+
```

Comment:
To save you from having to prepend `wave::` infront of your wave functions / classes. Plus all tests will be using the namespace `wave` so to be consistent I would advise you do the same even though you only use it once here.